topcoffea/modules/YieldTools.py

```diff
@@ -0,0 +1,424 @@
+import gzip
+import json
+import pickle
+import numpy as np
+from coffea import hist
+#from topcoffea.modules.HistEFT import HistEFT
+from topcoffea.modules.paths import topcoffea_path
+
+class YieldTools():
+
+    def __init__(self):
+
+        self.CAT_LST = ["cat_2lss_p","cat_2lss_m","cat_3l_1b_offZ_p","cat_3l_1b_offZ_m","cat_3l_2b_offZ_p","cat_3l_2b_offZ_m","cat_3l_1b_onZ","cat_3l_2b_onZ","cat_4l"]
+
+        self.PROC_MAP = {
+            "ttlnu" : ["ttW_centralUL17","ttlnu_private2017","ttlnuJet_privateUL17","ttlnuJet_privateUL18"],
+            "ttll"  : ["ttZ_centralUL17","ttll_TOP-19-001","ttllJet_privateUL17","ttllJet_privateUL18"],
+            "ttH"   : ["ttH_centralUL17","ttH_private2017","ttHJet_privateUL17","ttHJet_privateUL18"],
+            "tllq"  : ["tZq_centralUL17","tllq_private2017","tllq_privateUL17","tllq_privateUL18"],
+            "tHq"   : ["tHq_central2017","tHq_privateUL17"],
+            "tttt"  : ["tttt_central2017","tttt_privateUL17"],
+        }
```

Comment:
It might be useful to include a short comment description about what this and the other hard coded dictionaries are supposed to be used for. It's good to have a clear well defined purpose for what each of these variables are supposed to be used for and why they are structured the way they are. This is to avoid potentially duplicating data and bloating the code later on.

Reply:
Sounds like a good idea, added [here](https://github.com/TopEFT/topcoffea/commit/304f284608706a30289cf48c61d9397b40706594).